app/src/main/java/ch/epfl/sdp/blindwar/data/music/metadata/MusicMetadata.kt

```diff
@@ -0,0 +1,33 @@
+package ch.epfl.sdp.blindwar.data.music.metadata
+
+import ch.epfl.sdp.blindwar.game.model.Displayable
+
+abstract class MusicMetadata(var title: String = "",
```

Comment:
Maybe add kdoc for this class (you did this for most classes so you probably forgot here)